README.md

```diff
@@ -134,6 +135,19 @@ Separate each step of the task to an individual task. For example a "generate ic
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Use standard script names
```

Comment:
Add intro that aside from npm's predefined `npm start` and `npm test` aliases, a few script names have become standard in npm scripts. Namely `npm run build` and `npm run watch`.